<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>curve_show</class>
 <widget class="QDialog" name="curve_show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>511</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QCustomPlot" name="qCustomPlot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="House_Number">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>大棚编号：</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_right">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>3000000</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot\qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
